content/guides/models/tables/log_tables.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Log tables
+weight: 1
+---
+
+Visualize and log tabular data with W&B Tables. A table is a two-dimensional grid of data where each column has a single type of data. Each row represents one or more data points logged to a W&B [run]({{< relref "/guides/models/track/runs/" >}}). Tables support primitive and numeric types, as well as nested lists, dictionaries, and rich media types.
+
+Tables are a specialized [data type]({{< relref "/ref/python/data-types/" >}}) in W&B that are logged internally in W&B as [artifact]({{< relref "/guides/core/artifacts/" >}}) objects.
+
+You [create and log tables]({{< relref "#create-and-log-a-new-table" >}}) using the W&B Python SDK. When you create a table object, you specify the columns and data for the table and a [mode]({{< relref "#table-logging-modes" >}}). The mode determines how the table is logged and updated during your ML experiments.
```

Comment:
```suggestion
You [create and log Tables]({{< relref "#create-and-log-a-new-table" >}}) using the W&B Python SDK. When you create a Table, you specify its columns and data, along with a [mode]({{< relref "#table-logging-modes" >}}). The mode determines how the table is logged and updated by your ML experiments.
```
Again there are a few different ways we refer to this Table thing in a very short run.

Reply:
I worry that we are going to need to describe the Table panel, which may lead to prose like "A Table shows a table of....".

Reply:
We avoid using the product name in that case. E.g. We don't say "run a run..."